Release/src/CMakeLists.txt

```diff
@@ -127,10 +127,18 @@ endif()
 # Http client component
 if(CPPREST_HTTP_CLIENT_IMPL STREQUAL "asio")
   cpprest_find_boost()
-  cpprest_find_openssl()
-  target_compile_definitions(cpprest PUBLIC -DCPPREST_FORCE_HTTP_CLIENT_ASIO)
+  if(CPPREST_NO_OPENSSL)
+    cpprest_find_botan()
+    set(CPPREST_BOTAN_SSL_FLAG "-DCPPREST_BOTAN_SSL")
+    set(CPPREST_SSL_LINK_LIB cpprestsdk_botan_internal)
+  else()
```

Comment:
As mentioned above, this will not work unfortunately. We will still need OpenSSL for the listener.

Reply:
Makes sense!